sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -113,50 +107,9 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                                 }
                                 return true;
                             })
-                            .AddProcessor<ProfilingSessionTraceProcessor>()
-                            .AddAzureMonitorTraceExporter());
+                            .AddProcessor<ProfilingSessionTraceProcessor>());
 
-            builder.Services.ConfigureOpenTelemetryTracerProvider((sp, builder) =>
-            {
-                var azureMonitorOptions = sp.GetRequiredService<IOptionsMonitor<AzureMonitorOptions>>().Get(Options.DefaultName);
-                if (azureMonitorOptions.EnableLiveMetrics)
-                {
-                    // TODO: THIS WILL COME FROM THE EXPORTER
-
-                    //var manager = sp.GetRequiredService<LiveMetricsClientManager>();
-                    //builder.AddProcessor(new LiveMetricsActivityProcessor(manager));
-                }
-            });
-
-            builder.WithMetrics(b => b
-                            .AddHttpClientAndServerMetrics()
-                            .AddAzureMonitorMetricExporter());
-
-            builder.WithLogging(
-                    logging => logging.AddProcessor(sp =>
-                    {
-                        var azureMonitorOptions = sp.GetRequiredService<IOptionsMonitor<AzureMonitorOptions>>().CurrentValue;
-                        var azureMonitorExporterOptions = new AzureMonitorExporterOptions();
-                        azureMonitorOptions.SetValueToExporterOptions(azureMonitorExporterOptions);
-
-                        var exporter = new AzureMonitorLogExporter(azureMonitorExporterOptions);
-
-                        if (azureMonitorOptions.EnableLiveMetrics)
-                        {
-                            // TODO: THIS WILL COME FROM THE EXPORTER
-
-                            //var manager = sp.GetRequiredService<LiveMetricsClientManager>();
-
-                            //return new CompositeProcessor<LogRecord>(new BaseProcessor<LogRecord>[]
-                            //{
-                            //    new LiveMetricsLogProcessor(manager),
-                            //    new BatchLogRecordExportProcessor(exporter)
-                            //});
-                        }
-
-                        return new BatchLogRecordExportProcessor(exporter);
-                    }),
-                    options => options.IncludeFormattedMessage = true);
```

Comment:
Note to follow up: are we setting `IncludeFormattedMessage = true` in the Exporter?

Reply:
Confirmed:

https://github.com/Azure/azure-sdk-for-net/blob/2c27b4e8a1f93704f82a9ffe9a6067ac4fa42306/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/OpenTelemetryBuilderExtensions.cs#L76-L79